lib/quick-enemy.js

```diff
@@ -0,0 +1,14 @@
+const SimpleEnemy = require('./simple-enemy');
+
+
+var QuickEnemy = function (params) {
+  SimpleEnemy.call(this, params);
+  this.speed = 1.5;
+  this.health = 60;
```

Comment:
Love this, it's a great way to reuse the SimpleEnemy class so we don't have to repeat code. Also, I agree that a faster enemy should have less health. That's pretty standard for Tower Defense.